CIP-????/README.md

```diff
@@ -0,0 +1,471 @@
+---
+CIP: ????
+Title: Sums-of-products in Plutus Core
+Authors: Michael Peyton Jones <michael.peyton-jones@iohk.io>
+Status: Proposed
```

Comment:
... and unless @KtorZ suggests a convention otherwise I believe `Implementors:` should be set to those currently listed in the _Path to Active_.